lib/assets/stylesheets/modules/_technical-documentation.scss

```diff
@@ -1,137 +1,111 @@
-// Adds combination of margin and padding to headings to make them appear
-// correctly when they're linked to as anchors, and scale them on mobile
-// TODO: Headings are hidden underneath the sticky 'table of contents' on mobile
-// when following a link to them
-
-@mixin heading-offset($tabletTopMargin) {
-  // Scale margins with font size on mobile (16/19ths)
-  $mobileTopMargin: ceil($tabletTopMargin * (16 / 19));
-
-  padding-top: min($mobileTopMargin, $gutter-half);
-  margin-top: max(0, $mobileTopMargin - $gutter-half);
-
-  @include media(tablet) {
-    padding-top: min($tabletTopMargin, $gutter);
-    margin-top: max(0, $tabletTopMargin - $gutter);
-  }
-}
-
 .technical-documentation {
   // Required for IE11 to render <main> correctly
   display: block;
-  margin: 0 $gutter-half 10px;
+  margin: govuk-spacing(3);
+  font-size: 19px;
   max-width: 40em;
 
   line-height: 1.4;
 
-  color: $text-colour;
+  color:  $govuk-text-colour;
 
-  @include media(tablet) {
+    @include govuk-media-query(tablet) {
     // We add 2 pixels to the gutter here to ensure there's sufficient spacing between
     // an anchor link (see _anchored-heading.scss) on a heading and the left-hand pane.
-    margin: 0 ($gutter + 2px) $gutter;
+    margin: govuk-spacing(6) (govuk-spacing(6) + 2px);
   }
 
   .has-search-results-open & {
     visibility: hidden;
   }
 
-  // Headings
-  h1 {
-    @include bold-48;
-    @include heading-offset($gutter * 2);
-    border-top: 5px solid $text-colour;
+  > h1 {
```

Comment:
Because I used direct descendant selectors here to try and limit the scope, and because the list of servers in the API reference is inside a `div#server-list`, those server names and URLs end up unstyled:

<img width="1485" alt="Screen Shot 2019-09-04 at 11 14 32" src="https://user-images.githubusercontent.com/121939/64246784-5dfa9480-cf05-11e9-8fa3-2f726d0160c2.png">